src/ru/stepik/Main.java

```diff
@@ -1,8 +1,140 @@
 package ru.stepik;
 
+
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        task1();
+        task2();
+        task3();
+        task4();
+    }
+
+    public static void task1() {
+        int[] array = new int[3];
+        array[0] = 1;
+        array[1] = 2;
+        array[2] = 3;
+
+        double[] array1 = {1.57, 7.654, 9.986};
+        double sum1 = 0;
+
+        char[] array2 = new char[6];
+        array2[0] = 'S';
+        array2[1] = 'k';
+        array2[2] = 'y';
+        array2[3] = 'p';
+        array2[4] = 'r';
+        array2[5] = 'o';
+
+
+    }
+
+    public static void task2() {
+        int[] array = new int[3];
+        array[0] = 1;
+        array[1] = 2;
+        array[2] = 3;
+        int sum = 0;
+        for (int i = 0; i < array.length; i++) {
+            sum++;
+            if (array.length == sum) {
```

Comment:
sum переменная избыточна, мы можем проверять состояние переменной i
```suggestion
            if (i == array.length - 1) {
```